### PR TITLE
Feature/95293 new enviroments

### PR DESCRIPTION
**Pull request by DanOwenUKHO**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -41,50 +41,13 @@ variables:
     value: "ukhydrographicoffice/terraform-azure-powershell-unzip:1.3.7"
   - name: DeploymentPool
     value: "UKHO Ubuntu 1804"
-  - name: coverityPool
-    value: NautilusBuild
   - group: Covscan-vars
```
Comment (HugoBurgess) [inline azure-pipelines.yml:44]: This isn't needed now either. There's a separate PBI to remove the libraries from all the projects in Azure DevOps at a later date.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -877,29 +842,313 @@ stages:
...
+                  displayName: "Swap ESS API and ESS FulfilmentService Configuration AIOEnabled"
+                  condition: always()
+                  inputs:
+                    azureSubscription: "Exchange-Set-Service-vNext-IAT-A.011.08"
+                    scriptType: 'pscore'
+                    scriptLocation: 'scriptPath'
+                    scriptPath: "$(Build.SourcesDirectory)/terraformartifact/set_api_webjob_aio_feature_configuration.ps1"
+                    arguments: '-aiocells $(AioConfiguration.AioCells_FT) -aioenabled "true" -resourcegroup $(RESOURCE_GROUP_NAME) -webappname $(WEB_APP_NAME) -fulfilmentwebappsname $(fulfilmentWebAppsName)'
+
+                # - task: DotNetCoreCLI@2
```
Comment (HugoBurgess) [inline azure-pipelines.yml:963]: Can all the commented out tests be removed now?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -877,29 +842,313 @@ stages:
...
+        dependsOn:
+        - LiveDeployApp                 
+        pool: $(WindowPool)        
+        displayName: Post Deployment Actions
+        steps:
+          - template: Deployment/templates/retain-pipeline.yml
+
+  - stage: vnextiatDeploy
+    dependsOn:
+      - Devdeploy
```
Comment (HugoBurgess) [inline azure-pipelines.yml:874]: Does it need to wait for QC deploy too? Not sure what QC deploy is for!

File: azure-pipelines.yml [hunk 4]
```diff
@@ -877,29 +842,313 @@ stages:
...
+    displayName: vNext IAT Deploy (inc terraform, webapp deploy)
+    jobs:
```
Comment (HugoBurgess) [inline azure-pipelines.yml:876]: Waiting for branch name strategy from John to add condition here.

File: azure-pipelines.yml [hunk 5]
```diff
@@ -877,29 +842,313 @@ stages:
...
+      #         pwsh: true
+      #         targetType: filePath
+      #         filePath: '$(System.DefaultWorkingDirectory)/Deployment/ADDS-e2e-tests.ps1'
+      #         arguments: '-queueSecret $(QueueSecret) -pollSecret $(PollSecret) -pollTimeInSecs $(PollTimeInSecs) -functionEndPoint $(FunctionEndPoint)'
+          
+  - stage: vnexte2eDeploy
+    dependsOn: 
+      - vnextiatDeploy
+    displayName: vNext E2E Deploy (inc terraform, webapp deploy)
+    jobs:
```
Comment (HugoBurgess) [inline azure-pipelines.yml:1026]: Waiting for branch name strategy from John to add condition here.

File: azure-pipelines.yml (hunk 1 above)
Comment (DanOwenUKHO) [reply azure-pipelines.yml]: Removed

File: azure-pipelines.yml (hunk 2 above)
Comment (DanOwenUKHO) [reply azure-pipelines.yml:962]: Temporarily commented while environment data is being loaded

**Review by HugoBurgess — APPROVED** [review]
Looks good

**Pull request merged**